src/db/add_package.rs

```diff
@@ -86,7 +86,7 @@ pub(crate) fn add_package_into_database(
         &[
             &crate_id,
             &metadata_pkg.version,
-            &cratesio_data.release_time,
+            &cratesio_data.release_time.naive_utc(),
```

Comment:
What does `naive` mean here?

Reply:
`Naive`/`naive` stuff in `chrono` are times without time zones, and we store a timestamp without a timezone in the database